File: PairInArray.js
const number= [1,2,3,4,5,6]
const pair =()=>{
    let total=0;
    let curr;
    for(let i=0 ; i<number.length;i++){
        curr= number[i]
        for(let  j= i+1;j<number.length;j++){
            console.log(`(${curr} , ${number[j]})`)
            total++;
        }
        console.log()
    }
    console.log(`total pair :${total}`)
}

pair(number)

File: PairInArray.java

public class PairInArray {
     
public static void pair(int number[]){
    int totalPair=0;
  for(int i=0; i<number.length;i++) {
    int curr = number[i];
    for(int j =i+1;j<number.length;j++){
       System.out.print("(" +curr+","+number[j]+")"); 
       totalPair++;
    }
    System.out.println();
  }
  System.out.println("total pair :"+ totalPair);
}

    public static void main(String[] args) {
        int number[]= {2,4,6,8,10};
             pair(number);
    }
}
